src/plugins/EmojiTrigger.tsx

```diff
@@ -0,0 +1,87 @@
+import { InputRule } from "prosemirror-inputrules";
+import { Plugin } from "prosemirror-state";
+import Extension from "../lib/Extension";
+import { run } from "./BlockMenuTrigger";
+
+const OPEN_REGEX = /:([0-9a-zA-Z_]+)?$/;
```

Comment:
I think we need to account for ":+" and ":-" starts in the regex here for thumbs up/down